src/GraphQL.DataLoader.Tests/SimpleDataLoaderTests.cs

```diff
@@ -59,7 +59,7 @@ public async Task Operation_Can_Be_Cancelled()
             mock.Setup(store => store.GetAllUsersAsync(cts.Token))
                 .Returns(async (CancellationToken ct) =>
                 {
-                    await Task.Delay(200);
+                    await Task.Delay(5000, ct);
                     ct.ThrowIfCancellationRequested();
```

Comment:
If I understand this test correctly then it can be rewritten as 
```c#
            mock.Setup(store => store.GetAllUsersAsync(cts.Token))
                .Returns(async (CancellationToken ct) =>
                {
                    await Task.Delay(Timeout.Infinite, ct); // wait endlessly until the operation is canceled
                    return users;
                });
```

so no need to specify timeout - 200, 5000 or anything else.

Reply:
You probably could, but I wouldn’t unless we have continuing problems, or else upon failure it would hang forever

Reply:
We could go longer like a minute 60000. There should be no point to waiting longer than that.

Reply:
Updated to 60000ms

Reply:
I understand that setting an endless timeout is dangerous. I just wanted to say that the algorithm in fact should always end with the cancellation of the operation, so you can wait endlessly inside the method. I'm OK with 5000 or 60000.

Reply:
Let's leave it longer.  I don't know how long appveyor might hang for, but in the debug environment, you can always just stop execution.  So a longer timeout should help tests pass.  Hopefully it will never hang that long, but a minute isn't a problem for appveyor.